### DISCUSSION TITLE
Including Avahi into the core-image-sato image

### DISCUSSION THREAD

**Discussion by VishwanathB45** [OE4T/meta-tegra#1409] [Q&A] [answered]
Hi,
I'm using kirkstone-l4t-r32.7.x branch in meta-tegra.I want to add the avahi into the core-image-sato image.
when I added the avahi in my local.conf like IMAGE_INSTALL:append = "avahi" ,
I am getting an error: No match for argument: avahi ,Error: Unable to find a match: avahi
Could you kindly share any insights or tips on how to accomplish this?

**Comment by dwalkes** [comment]
You should include a leading space in `" avahi"` - see https://docs.yoctoproject.org/4.0.13/dev-manual/customizing-images.html#customizing-images-using-local-conf - although this doesn't look like the error message I'd expect in this case.  I wonder if there might be some other issue in your local.conf.

**Comment by dwalkes** [comment, marked as answer]
The error is during the `dnf` step, and I think this is specific to the `avahi` recipe since it [doesn't put files into PN (the package name)](https://github.com/yoctoproject/poky/blob/d8d6d921fad14b82167d9f031d4fca06b5e01883/meta/recipes-connectivity/avahi/avahi_0.8.bb#L121).  I think you need something like `IMAGE_INSTALL:append = " avahi-daemon" ` instead.
